recipe/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1000
 
 requirements:
   build:
```

Comment:
No `host` section means that we need to run `conda smithy update-cb3` here first.